Makefile

```diff
@@ -4,21 +4,29 @@ TEMP_DIR:=$(shell mktemp -d)
 ARCH?=amd64
 OUT_DIR?=./_output
 
+OPENAPI_PATH=$(GOPATH)/src/k8s.io/kube-openapi
+
 VERSION?=latest
 
+ifeq  ($(PLATFORM),Linux)
+SED_REPLACE :=
+else
+SED_REPLACE := ''
+endif
+
 .PHONY: all build-test-adapter test verify-gofmt gofmt verify test-adapter-container
 
 all: build-test-adapter
 build-test-adapter: vendor
-	CGO_ENABLED=0 GOARCH=$(ARCH) go build -o $(OUT_DIR)/$(ARCH)/test-adapter github.com/kubernetes-incubator/custom-metrics-apiserver/test-adapter
+	CGO_ENABLED=0 GOOS=linux GOARCH=$(ARCH) go build -o $(OUT_DIR)/$(ARCH)/test-adapter github.com/kubernetes-incubator/custom-metrics-apiserver/test-adapter
```

Comment:
`go build`  should actually be run with `-mod vendor` as otherwise `./vendor` won't be used afaik.